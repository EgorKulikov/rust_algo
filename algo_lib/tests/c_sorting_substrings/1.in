abacaba
6
4 7
1 2
1 1
3 4
5 7
1 3
