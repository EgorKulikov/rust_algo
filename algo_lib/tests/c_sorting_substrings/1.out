1 1
1 2
1 3
5 7
3 4
4 7
